assets/js/blocks/rating-filter/block.tsx

```diff
@@ -272,7 +278,11 @@ const RatingFilterBlock = ( {
 		[ checked ]
 	);
 
-	if ( ! filteredCountsLoading && displayedOptions.length === 0 ) {
+	if (
+		! isEditor &&
```

Comment:
Why do we need to check for `isEditor` here? In the editor, the `displayedOptions` will be set to preview data if the store has no ratings, so I don't think we need to check for `isEditor`.

Reply:
True that. Earlier I was exploring a different approach for displaying the notice (where the `displayedOptions` wasn't being set to preview data), that's the reason why it was introduced in the first place: updated.